src/ASM/ASMs2D.C

```diff
@@ -155,16 +154,14 @@ bool ASMs2D::read (std::istream& is)
   if (!is.good() && !is.eof())
   {
     std::cerr <<" *** ASMs2D::read: Failure reading spline data"<< std::endl;
-    delete surf;
-    surf = nullptr;
+    surf.reset();
     return false;
   }
   else if (surf->dimension() < 2)
   {
     std::cerr <<" *** ASMs2D::read: Invalid spline surface patch, dim="
-	      << surf->dimension() << std::endl;
-    delete surf;
-    surf = nullptr;
+          << surf->dimension() << std::endl;
```

Comment:
Wrong indent

Reply:
fixed. my IDE replaced a tab with spaces but my tabs are 2, not 8..